### PR TITLE
Add runtime variables to bpa chart

### PR DESCRIPTION
**Pull request by schlagtim**
Add runtime variables to charts:
- SIDEBAR_CLOSE_ON_STARTUP
- SIDEBAR_HIDE_BURGER_BUTTON

Signed-off-by: Tim Schlagenhaufer <tim.schlagenhaufer@bosch.io>

### CODE REVIEW

File: charts/bpa/values.yaml [hunk 1]
```diff
@@ -66,6 +66,13 @@ bpa:
     i18n:
       locale: en
       fallbackLocale: en
+    runtimeVariables:
```
Comment (etschelp) [inline charts/bpa/values.yaml:69]: I think naming this runtime variables is confusing as all entries here are. I would prefer something like _bpa.config.frontend.closeSidebar_

File: charts/bpa/values.yaml [hunk 2]
```diff
@@ -66,6 +66,13 @@ bpa:
...
+      # -- Enables runtime variables which are injected into the frontend code on container startup (only if at least one runtime variable value has been set). These function e.g. as feature toggles for specific frontend changes. NOTE: The container startup time is impacted by using these.
+      enabled: false
```
Comment (etschelp) [inline charts/bpa/values.yaml:71]: This is not needed as you can check the property directly in the configmap

File: charts/bpa/templates/bpa_configmap.yaml [hunk 3]
```diff
@@ -35,3 +35,9 @@ data:
 {{- if .Values.bpa.config.privacyPolicy.enabled }}
   BPA_PRIVACY_POLICY_URL: {{ tpl .Values.bpa.config.privacyPolicy.urlOverride . | default (printf "https://bpa%s/privacyPolicy" .Values.global.ingressSuffix) | quote }}
 {{- end }}
+{{- if .Values.bpa.config.runtimeVariables.enabled }}
```
Comment (etschelp) [inline charts/bpa/templates/bpa_configmap.yaml:38]:
```
{{- if .Values.bpa.config.frontend.sidebarClosed }}
SIDEBAR_CLOSE_ON_STARTUP: {{ .Values.bpa.config.frontend.sidebarClosed | quote }}
```
Is easier

Comment (schlagtim) [reply charts/bpa/templates/bpa_configmap.yaml]: Done

File: charts/bpa/values.yaml (hunk 2 above)
Comment (schlagtim) [reply charts/bpa/values.yaml]: Done

File: charts/bpa/values.yaml (hunk 1 above)
Comment (schlagtim) [reply charts/bpa/values.yaml]: Done

**Pull request merged**